tests/tiff/tiff_dpi.c

```diff
@@ -0,0 +1,77 @@
+#include "gd.h"
```

Comment:
could you add a comment block to the top of the file giving an overview of the test ?